### PR TITLE
[circle2circle] Fuse add with tconv

### PR DESCRIPTION
**Pull request by jinevening**
This adds --fuse_add_with_tconv option

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Draft PR: #4375

### CODE REVIEW

**Review by mhs4670go — APPROVED** [review]
LGTM

File: compiler/circle2circle/src/Circle2Circle.cpp [hunk 1]
```diff
@@ -130,6 +136,8 @@ int entry(int argc, char **argv)
     options->enable(Algorithms::ResolveCustomOpBatchMatMul);
     options->enable(Algorithms::ResolveCustomOpMatMul);
   }
+  if (arser.get<bool>("--fuse_add_with_tconv"))
+    options->enable(Algorithms::FuseAddWithTConv);
```
Comment (seanshpark) [inline compiler/circle2circle/src/Circle2Circle.cpp:140]: Is it possible to add also to `--all` ?

Comment (seanshpark) [reply compiler/circle2circle/src/Circle2Circle.cpp:140]: @mhs4670go , was there any particular reason for `fuse_batchnorm_with_tconv` not added to `all` ?

Comment (mhs4670go) [reply compiler/circle2circle/src/Circle2Circle.cpp:140]: @seanshpark When we were discussing the pass, we decided to add it to `--all` section as the path became more mature. There are still few places to use this path, and I'm gonna put it in after some more tests are added, which by the way, is the reason why I don't close related issue yet.

Comment (seanshpark) [reply compiler/circle2circle/src/Circle2Circle.cpp:140]: OK, thanks!

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**